app/src/main/java/com/example/dine_and_donate/LoginActivity.java

```diff
@@ -24,31 +24,31 @@
 import com.google.firebase.database.FirebaseDatabase;
 import com.google.firebase.database.ValueEventListener;
 
-import java.util.HashMap;
-import java.util.Map;
-
 public class LoginActivity extends AppCompatActivity {
```

Comment:
can you revert all the changes to this file? it doesn't seem like they are necessary (maybe just a result of merge conflicts)